JanePhotoBrowser/PhotoBrowser/PhotoBrowserFullscreen/PhotoBrowserStyleKit.swift

```diff
@@ -58,5 +63,18 @@ public class PhotoBrowserIconography: NSObject {
         
         return imageOfXIcon!
     }
-
+    
+    public static func imageOfZoomInIcon() -> UIImage? {
+        if #available(iOS 13, *) {
+            return UIImage(systemName: "arrow.up.left.and.arrow.down.right")
```

Comment:
When in a pod you must specify the bundle or it will not work once you try to integrate with the app